connector/oidc/oidc.go

```diff
@@ -44,6 +44,9 @@ type Config struct {
 	// InsecureEnableGroups enables groups claims. This is disabled by default until https://github.com/dexidp/dex/issues/1065 is resolved
 	InsecureEnableGroups bool `json:"insecureEnableGroups"`
 
+	// Skips checking whether the requested domain in the Login Callback matches the configured Issuer
+	SkipIssuerCallbackDomainCheck bool `json:"skipIssuerCallbackDomainCheck"`
```

Comment:
By convention I think this should be an `insecure` option, but that only really matters for upstreaming